code/controllers/subsystem/stats_collector.dm

```diff
@@ -0,0 +1,17 @@
+/// Collects simple round statistics periodically
+SUBSYSTEM_DEF(stats_collector)
+	name      = "Round Stats"
+	wait      = 30 SECONDS
+	priority  = SS_PRIORITY_PAGER_STATUS
+	runlevels = RUNLEVELS_DEFAULT | RUNLEVEL_LOBBY
+	flags     = SS_KEEP_TIMING
```

Comment:
shouldn't this be indents between the vars and =s

Reply:
those aren't code indent, it's just spaces for alignment
tabs unfortunately don't work for alignment like this (despite it being what they were meant to be for..) since different viewers/platforms will display them as 2/4/8 spaces
i just took the habit of aligning multi assigns like that for readability